examples/example1.c

```diff
@@ -53,7 +53,7 @@ int main() {
     if (!rdbToJson) goto PARSER_ERROR;
 
     /* Filter keys that starts with the word `mylist` */
-    filterKey = RDBX_createHandlersFilterKey(parser, "mylist.*", 0 /*flags*/);
+    filterKey = RDBX_createHandlersFilterKey(parser, "mylist.*", 0 /*exclude*/);
```

Comment:
Side note, example code crashes (with the current master). Maybe we should add it to CI.

Reply:
fixed. I will add it to tests. Thanks.